lib/rubocop/config_loader.rb

```diff
@@ -220,6 +222,19 @@ def check_duplication(yaml_code, absolute_path)
         end
       end
 
+      def check_enabled_value(hash, parent = nil)
+        hash.each do |key, value|
+          check_enabled_value(value, key) if value.is_a?(Hash)
+
+          next unless key == 'Enabled' && value.class == String
+
+          warn Rainbow(
+            "Cop `#{parent}` is enabled by value '#{value}' specified. " \
+            'It may unexpected behavior unless set true or false.'
```

Comment:
I think we should probably raise an error instead and just say that property X of cop Y is supposed to be a boolean and it's not.

Reply:
OK. I change to raise an exception.

Reply:
Like this?

![Screenshot from 2019-08-10 20-05-44](https://user-images.githubusercontent.com/4487291/62821104-4819de00-bbaa-11e9-8828-db46c5ec486a.png)

Reply:
Actually instead of exception this can just be an error message with some error code. I guess the stack trace will just confuse most users. 

It will also be a good idea to colour the names of the cop and the property in the error message and to mention its current value that's causing the error.

Reply:
How is this?

![Screenshot from 2019-08-10 21-51-25](https://user-images.githubusercontent.com/4487291/62822043-06dcfa80-bbb9-11e9-81b1-ce52257e8388.png)

Reply:
It's good.